napari_skimage_regionprops/_parametric_images.py

```diff
@@ -1,7 +1,87 @@
 import numpy as np
 from napari_tools_menu import register_function
 import numpy
+from deprecated import deprecated
+from napari.utils import notifications
 
+@register_function(menu="Measurement maps > Measurements on labels with map array (nsr)")
+@register_function(menu="Visualization > Measurements on labels with map array (nsr)")
+def visualize_measurement_on_labels_with_map_array(labels_layer:"napari.layers.Labels", column:str = "label", viewer:"napari.Viewer" = None) -> "napari.types.ImageData":
```

Comment:
The name of the function is a bit cumbersome. How about `map_measurements_on_labels`?

Reply:
I love it! 😻